index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">

```

Comment:
wyciagnalem nav poza header. 
Tzn moglby byc tutaj tez header, ale stwierdzilem ze skoro ten header ma miec tylko nav, to jest nie potrzebny - i niech to bedzie po prostu nav - jako caly pasek na gorze strony

wazne bylo aby oddzielic go od tego bannerka pod nim - sa teraz obok siebie a nie jedno w drugim